3f/pygmentize/2016-11-05.yaml

```diff
@@ -0,0 +1,9 @@
+title: Remote Code Execution
+link: https://github.com/dedalozzo/pygmentize/issues/1
+cve: ~
+branches:
+    master:
+        time: 2016-12-31 00:00:00
+        versions: ['<=1.1']
```

Comment:
I was a bit over-zealous in my fork pruning a few months ago and removed this fork. I'll close this PR and create a new one tonight.